### PR TITLE
Enable Avatar Hand Touch Script by Default

### PR DESCRIPTION
**Pull request by luiscuenca**
https://highfidelity.manuscript.com/f/cases/14958/Enable-Avatar-Hand-Touch-Script-by-Default

This PR loads the script system/constrollers/handTouch.js by default

### CODE REVIEW

**Comment by hyperlogic** [comment]
https://highfidelity.manuscript.com/f/cases/14958/Enable-Avatar-Hand-Touch-Script-by-Default

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9784/HighFidelity-Beta-PR13171-acd1721c88376575d289d3e81f01cc7f793a4353.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9784/HighFidelity-Beta-PR13171-acd1721c88376575d289d3e81f01cc7f793a4353.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9784/HighFidelity-Beta-Interface-PR13171-acd1721c88376575d289d3e81f01cc7f793a4353.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9784/HighFidelity-Beta-PR13171-acd1721c88376575d289d3e81f01cc7f793a4353.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9784/HighFidelity-Beta-Interface-PR13171-acd1721c88376575d289d3e81f01cc7f793a4353.exe

File: scripts/system/controllers/controllerScripts.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ var CONTOLLER_SCRIPTS = [
     "controllerDisplayManager.js",
     "grab.js",
     "toggleAdvancedMovementForHandControllers.js",
+	"handTouch.js",
```
Comment (hyperlogic) [inline scripts/system/controllers/controllerScripts.js]: Indentation?

**Comment by hyperlogic** [comment]
Also, handTouch.js is not eslint clean and has lots of dangling whitespace, can we clean that up, please?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10422/HighFidelity-Beta-PR13171-df32fe6.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10422/HighFidelity-Beta-PR13171-df32fe6.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10422/HighFidelity-Beta-Interface-PR13171-df32fe6.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10422/HighFidelity-Beta-PR13171-df32fe6.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10422/HighFidelity-Beta-Interface-PR13171-df32fe6.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10533/HighFidelity-Beta-PR13171-9126a6f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10533/HighFidelity-Beta-PR13171-9126a6f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10533/HighFidelity-Beta-Interface-PR13171-9126a6f.dmg

**Comment by hyperlogic** [comment]
QA Test plan:
1) first run the test plan from this earlier pr #12071
2) run the Impact of Finger Touching Test plan from testrail [here](https://highfidelity.testrail.net/index.php?/cases/view/11448)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10533/HighFidelity-Beta-PR13171-9126a6f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10533/HighFidelity-Beta-Interface-PR13171-9126a6f.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Pull request merged**